src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.main__container {
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  border: 1px solid $color-gray;
```

Comment:
Ensure that the `$color-gray` variable is defined in your SCSS files. If it's not defined, this could lead to a compilation error.